fixProc.js

```diff
@@ -10,11 +10,11 @@ module.exports = function (okey, password) {
   var decrypted;
   if (!match) {
     var match2 = key.match(fullRegex);
-    decrypted = new Buffer(match2[2].replace(/\n\r?/g, ''), 'base64');
+    decrypted = new Buffer(match2[2].replace(/\r?\n/g, ''), 'base64');
   } else {
     var suite = 'aes' + match[1];
     var iv = new Buffer(match[2], 'hex');
-    var cipherText = new Buffer(match[3].replace(/\n\r?/g, ''), 'base64');
```

Comment:
I don't really have access to an operating system that puts in those types of returns so I'd need to figure out how to create one, probably need to make sure to store it as a buffer to prevent git from 'fixing' it.